src/lifetime-elision.md

```diff
@@ -7,8 +7,10 @@ compiler can infer a sensible default choice.
 
 In order to make common patterns more ergonomic, Rust allows lifetimes to be
 *elided* in [function item], [function pointer] and [closure trait] signatures.
-The following rules are used to infer lifetime parameters for elided lifetimes.
-It is an error to elide lifetime parameters that cannot be inferred.
+An elided lifetime can be specified either by omitting it, or by using `'_`.
+For lifetimes in paths, using `'_` is preferred. The following rules are used
```

Comment:
A few things here:

- What is a lifetime in a path? Do you mean as a parameter?
- Should probably mention that omitting a lifetime in a trait object type is equivalent to `'static`, not `'_`.

Reply:
Also, is `'_` not preferred for references as well? I thought omitting lifetimes was going to be deprecated.